math/odd_or_even.cpp

```diff
@@ -0,0 +1,53 @@
+/**
+ * @file
+ * \brief Program to check if a number is an odd or even number.
+ *
+ * \details
+ * if a number is odd its remainder when divided by 2 is not zero. if the remainder is zero,
+ * then it is an even number
+ * @author nathalieleave
+ */
+#include <cassert>
+#include <cmath>
+#include <iostream>
```

Comment:
Please add a one-line description of what the library/header is for (**see the example below**).

```c++
#include <iostream>    /// for IO operations
#include <cassert>    /// for assert
```